clients/instance/ibm-pi-placement-groups.go

```diff
@@ -12,19 +12,19 @@ import (
 	"github.com/IBM-Cloud/power-go-client/power/models"
 )
 
-//IBMPIPlacementGroupClient ...
+//IBMPIPlacementGroupClient
 type IBMPIPlacementGroupClient struct {
 	IBMPIClient
 }
 
-// NewIBMPIPlacementGroupClient ...
+// NewIBMPIPlacementGroupClient
 func NewIBMPIPlacementGroupClient(ctx context.Context, sess *ibmpisession.IBMPISession, cloudInstanceID string) *IBMPIPlacementGroupClient {
 	return &IBMPIPlacementGroupClient{
 		*NewIBMPIClient(ctx, sess, cloudInstanceID),
 	}
 }
 
-// Get PI Placementgroup
+// Get a Placement Group
```

Comment:
There are going to be more than one type of placement groups soon we should keep the type of placement group which in this case is Instance